recipe/meta.yaml

```diff
@@ -37,7 +37,6 @@ requirements:
     - python
     - setuptools
     - packaging
-    - {{ pin_compatible('numpy') }}
```

Comment:
It's probably brought in by one of the other dependencies, but I think we should still include it as a runtime dependency